frontend/src/scenes/settings/project/ProjectSettings.tsx

```diff
@@ -141,6 +142,21 @@ export function ProjectVariables(): JSX.Element {
                 </p>
                 <CodeSnippet thing="project ID">{String(currentTeam?.id || '')}</CodeSnippet>
             </div>
+            {['https://eu.posthog.com', 'https://us.posthog.com'].includes(
+                preflightLogic.values.preflight?.site_url ?? ''
```

Comment:
I think there is a `region` property on the preflight logic that is way better suited to this (as then you are just checking if it is set rather than url checks

Also please use the hooks instead of directly accessing the logic so:
```ts
const { preflight } = useValues(preflightLogic)
const { region } = preflight

// and then later

{region ? <div>{region.toUppercase()} Cloud</div> : null}
```

Reply:
![2024-01-16 at 16 20 55@2x](https://github.com/PostHog/posthog/assets/13001502/e9294c05-3c08-44e6-858e-3291a5eabc7d)
the region has to be uppercase according to typescript